.circleci/config.yml

```diff
@@ -124,6 +124,7 @@ jobs:
           command: |
             yarn task --task compile --start-from=auto --no-link --debug
             git diff --exit-code
+            yarn dedupe --check
```

Comment:
Yeah, this will fail when there are packages that can be de-duplicated.

Part of the perf regression was that we bundled different version of `chai` and aria-query` which could be de-duplicated.